### PR TITLE
[androiddebugbridge] Fix volume channel for android 11/12

### PR DESCRIPTION
**Pull request by GiviMAD**
Device "ChromeCast with Google TV" was recently updated to android 12.
I found out that volume channel is not working anymore (neither should work on android 11), it is due the 'media' cli removal on android 11.
This PR aims to fix the volume channel for both versions (just tested on android 12).

Regards!

Signed-off-by: Miguel Álvarez <miguelwork92@gmail.com>

### CODE REVIEW

**Comment by GiviMAD** [comment]
I'll add another commit, I have discovered how to get the max volume level on android 12, can be gathered from the device properties

**Comment by GiviMAD** [comment]
Now it's ready. I left the deviceMaxVolume config in case other device miss this property.

File: bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -58,6 +58,30 @@
 				<description>JSON config that allows to modify the media state detection strategy for each app. Refer to the binding
 					documentation.</description>
 			</parameter>
+			<parameter name="volumeSettingKey" type="text" required="true">
```
Comment (lolodomo) [inline bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml:61]: As there is a default value provided, this parameter is not "required".

File: bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -58,6 +58,30 @@
...
+					<option value="volume_music_hdmi">volume_music_hdmi</option>
+					<option value="volume_music_headphone">volume_music_headphone</option>
+					<option value="volume_music_headset">volume_music_headset</option>
+					<option value="volume_music_headset">volume_music_headset</option>
+					<option value="volume_music_usb_headset">volume_music_usb_headset</option>
+					<option value="volume_system">volume_system</option>
+				</options>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="deviceMaxVolume" type="integer" required="true" min="1" max="100">
```
Comment (lolodomo) [inline bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml:79]: As there is a default value provided, this parameter is not "required".

File: bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml [hunk 3]
```diff
@@ -58,6 +58,30 @@
...
+				<options>
+					<option value="volume_bluetooth_sco">volume_bluetooth_sco</option>
+					<option value="volume_music">volume_music</option>
+					<option value="volume_music_bt_a2dp">volume_music_bt_a2dp</option>
+					<option value="volume_music_hdmi">volume_music_hdmi</option>
+					<option value="volume_music_headphone">volume_music_headphone</option>
+					<option value="volume_music_headset">volume_music_headset</option>
+					<option value="volume_music_headset">volume_music_headset</option>
+					<option value="volume_music_usb_headset">volume_music_usb_headset</option>
+					<option value="volume_system">volume_system</option>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml:75]: If you define options, at least some of the options should have a label different from its value. If not, the options are useless.
I would suggest to not use _ inside the labels.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml [hunk 4]
```diff
@@ -58,25 +58,24 @@
...
-					<option value="volume_music_usb_headset">volume_music_usb_headset</option>
-					<option value="volume_system">volume_system</option>
+					<option value="volume_bluetooth_sco">volume bluetooth sco</option>
+					<option value="volume_music">volume music</option>
+					<option value="volume_music_bt_a2dp">volume music bt a2dp</option>
+					<option value="volume_music_hdmi">volume music hdmi</option>
+					<option value="volume_music_headphone">volume music headphone</option>
+					<option value="volume_music_headset">volume music headset</option>
+					<option value="volume_music_usb_headset">volume music usb headset</option>
+					<option value="volume_system">volume system</option>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml:74]: You have also to adjust again the i18n entries.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml (hunk 4 above)
Comment (GiviMAD) [reply bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml:74]: ty!

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**